dotcom-rendering/cypress/plugins/index.js

```diff
@@ -7,16 +7,38 @@
 // You can read more here:
 // https://on.cypress.io/plugins-guide
 // ***********************************************************
+import path from 'path';
 
 const webpackPreprocessor = require('cypress-webpack-preprocessor-v5');
 
 module.exports = (on, config) => {
 	config.env = { ...config.env, ...process.env };
 
 	const webpackConfig = webpackPreprocessor.defaultOptions;
-	webpackConfig.webpackOptions.module.rules[0].exclude =
+	// Adding this here so that we can import the fixture in a cypress test
+	webpackConfig.webpackOptions.resolve = {
```

Comment:
I think I might have a go and testing this locally since I think it still _might_ be trying to compile the entire project?

Reply:
Okay having a play around with this it does seem like it's only compiling the the fixture 👍